listener/upload_listener.py

```diff
@@ -45,7 +45,6 @@
 DATABASE_ERROR = Counter('ve_listener_database_error', '# of database errors')
 UPLOAD_NO_RPMDB = Counter('ve_listener_upl_no_rpmdb', '# of systems ignored due to missing rpmdb')
 MESSAGE_PARSE_ERROR = Counter('ve_listener_message_parse_error', '# of message parse errors')
-NEW_REPO = Counter('ve_listener_upl_new_repo', '# of new repos inserted')
```

Comment:
I removed this counter because I think its useless now, since repos get synced from vmaas.